api/WindowOrWorkerGlobalScope.json

```
@@ -1357,6 +1357,66 @@
             }
           }
         }
+      },
+      "structuredClone": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/structuredClone",
+          "spec_url": "https://html.spec.whatwg.org/multipage/structured-data.html#dom-structuredclone",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "deno": {
+              "version_added": "1.13",
+              "partial_implementation": true,
+              "notes": [
+                "The <code>message</code> parameter does not support cloning <code>Blob</code> values.",
+                "The <code>transfer</code> parameter does not accept <code>ArrayBuffer</code> items. Passing an <code>ArrayBuffer</code> results in an error being thrown."
+              ]
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "nodejs": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
```

Generally, we need two distinct implementations to set experimental to `true`. Check out [the data guideline](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status) for _all_ the nuance.

```suggestion
            "experimental": true,
```

---

Whoops. Fixed it now. This is what I get for copy pasting :sweat_smile:

---

Happens to all of us!